tests/autoyast/validate_salt_formula.pm

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Make sure we are the salt formula has set message-of-the-day(motd)
+# Maintainer: ybontakis <ybonatakisz@suse.com>
```

Comment:
s/ybonatakisz/ybonatakis/ if it is not a typo and it is some issue about privacy, let's discuss offline better ;)